src/Redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,16 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const categoriesSlice = createSlice({
+  name: 'categories',
+  initialState: {
+    items: [],
+    status: 'Under construction',
+  },
+  reducers: {
+    checkstate: (state) => {
+      state.status = 'Under construction';
+    },
+  },
```

Comment:
- [ ] Well done 👏🏼  implementing the categories slice, however as stated in the project requirement in the screenshot below:
![image](https://user-images.githubusercontent.com/68030297/236914258-b1e834f5-cf78-497e-a8a4-e078b53e3b82.png)

So if the action.payload that is coming from the component is "Under construction" then return it, otherwise return the state which in your case is `state.status` 👍🏼